### PR TITLE
Feature/avocato

### PR DESCRIPTION
**Pull request by guevarajoge**
1 + 2 = ?

### CODE REVIEW

**Review by sarahverbelen — APPROVED** [review]
looking good!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <h1>Avocato 2</h1>
```
Comment (eliseboes) [inline index.html:8]: Change to Advocado

Comment (wienertgillijns) [reply index.html:8]: Perfect!

Comment (wienertgillijns) [inline index.html:8]: Very good description

**Review by wienertgillijns — APPROVED** [review]

**Pull request closed without merging**